CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+## 0.21.0-rc.0 / 2020-06-02
+
+* [CHANGE] [amtool] Remove default assignment of environment variables. #2161
+* [CHANGE] Enforce 512KB event size limit for Pagerduty events. #2225
+* [ENHANCEMENT] [Slack] Add channel when logging errors. #2177
+* [ENHANCEMENT] [Cluster] Log error on reconnect failures. #2260
+* [ENHANCEMENT] [amtool] Add `cluster` command to show cluster and peer statuses. #2256
+* [ENHANCEMENT] Add redirection from `/` to the routes prefix when it isn't empty. #2235
+* [ENHANCEMENT] Add `retry` field when logging notification failures. #2188
+* [ENHANCEMENT] [API v2] Log request path and method. #2261
+* [ENHANCEMENT] [Webhook] Add `max_alerts` option to limit the number of alerts included in the payload. #2274
```

Comment:
FEATURE

Reply:
It's a relatively minor thing to an existing feature, I'd say enhancement personally.

Reply:
Ok, it is also true that it is kinda edge-case-y